### PR TITLE
Cpm turn state

### PR DESCRIPTION
**Pull request by 2Percent-Milk**
AJ  and I made the turn state

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -13,5 +13,68 @@
     "**/.settings": true,
```
Comment (witcpalek) [inline .vscode/settings.json]: Don't need the settings.json file

File: .wpilib/wpilib_preferences.json [hunk 2]
```diff
@@ -2,5 +2,5 @@
     "enableCppIntellisense": true,
     "currentLanguage": "cpp",
     "projectYear": "2020",
-    "teamNumber": 302
+    "teamNumber": 1
```
Comment (witcpalek) [inline .wpilib/wpilib_preferences.json]: Don't need the wpilib_preferences.json

File: src/main/cpp/controllers/controlpanel/ControlPanelTurn.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+void ControlPanelTurn::Init()
+{
+    m_target = m_control_panel_manipulator -> GetCurrentPosition() + m_offset;
+    m_control_panel_manipulator -> SetControlConstants(m_control_data);
+}
+
+void ControlPanelTurn::Run()
+{
+    m_control_panel_manipulator -> SetOutput(m_control_data -> GetMode(), m_target);
+    m_current_position = m_control_panel_manipulator -> GetCurrentPosition();
```
Comment (witcpalek) [inline src/main/cpp/controllers/controlpanel/ControlPanelTurn.cpp]: Why are you stowing the current position?  You don't use it here and where you need it, you make this call again.

File: src/main/cpp/test/XboxDisplay.cpp [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#include <test/XboxDisplay.h>
+#include <Test/XboxDisplay.h>
```
Comment (witcpalek) [inline src/main/cpp/test/XboxDisplay.cpp]: Don't need XboDisplay.h

Also, add documentaton to ControlPanelTurn.h/.cpp

Comment (2Percent-Milk) [reply src/main/cpp/test/XboxDisplay.cpp]: I actually do need to include it or it wont build

**Pull request closed without merging**